scripts/make_summary_perfect.py

```diff
@@ -514,9 +508,7 @@ def calculate_weighted_prices(n, label, weighted_prices):
             if names.empty:
                 continue
 
-            load += (
-                n.links_t.p0[names].groupby(n.links.loc[names, "bus0"], axis=1).sum()
-            )
+            load += n.links_t.p0[names].T.groupby(n.links.loc[names, "bus0"]).sum()
```

Comment:
Just double-checking the transposes here:

So far, in most cases `.groupby(..., axis=1).sum()` is replaced by `.T.groupby(...).sum().T`